virtManager/details/details.py

```diff
@@ -1967,6 +1965,11 @@ def remove_device(self, devobj):
             buttons=Gtk.ButtonsType.OK,
             dialog_type=Gtk.MessageType.INFO)
 
+    # Device removal
+    def remove_device(self, devobj):
+        success = self.remove_devobj_internal(devobj)
+        if success:
+            self.disable_apply()
```

Comment:
You removed the newline after the function. Please fix this in the next version. I suggest visually reviewing each diff before you submit things, the changes will stand out more